### ISSUE TITLE
ML - Training Script - Linear Regression

### ISSUE THREAD

**Issue by Mohammad4844** [COSC481W-2024Fall/MLite#6]
## User Story
As a user, I would like to train a linear regression model. I want to be able to go to my dataset, select a numerical label, and train a Linear Regression model.

### Persona
Existing users looking to train a model.

### Feature
Be able to run a training job through our app, specifically for a linear regression model.

### Business Value
Users can train a new model type from our app for numerical datasets and regression problems.

## Tasks

1. Script takes in input hyperparameters
2. Script loads the dataset
3. Script initializes a model & trains on the dataset
4. Script saves/uploads the trained model

## Acceptance Criteria
- Will NOT implement integration with our Rails app. Simply ML scripts to train a given model.

## Acceptance Tests

- Users can schedule a Linear Regression model training
- Training runs successfully
- Final model is saved in an appropriate location
